File: Iniciante/URI 1041.pas
var
	x : double;
	y : double;

begin
    readLn(x, y);
	if ((x = 0) and (y = 0)) then
		writeln('Origem')
	else if (x = 0) then
		writeln('Eixo Y')
	else if (y = 0) then
		writeln('Eixo X')
	else if ((y > 0) and (x > 0)) then
		writeln('Q1')
	else if ((y > 0) and (x < 0)) then
		writeln('Q2')
	else if ((y < 0) and (x < 0)) then
		writeln('Q3')
	else if ((y < 0) and (x > 0)) then
		writeln('Q4')
end.

File: Iniciante/URI 1041.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
x = as.double(line[[1]][1])
y = as.double(line[[1]][2])

if (x == 0 && y == 0) {
    write(sprintf("Origem"), '')
} else if (x == 0) {
    write(sprintf("Eixo Y"), '')
} else if (y == 0) {
    write(sprintf("Eixo X"), '')
} else if (y > 0 && x > 0) {
    write(sprintf("Q1"), '')
} else if (y > 0 && x < 0) {
    write(sprintf("Q2"), '')
} else if (y < 0 && x < 0) {
    write(sprintf("Q3"), '')
} else if (y < 0 && x > 0) {
    write(sprintf("Q4"), '')
}